docs/source/admin/quick_howto/server_capabililty.rst

```diff
@@ -0,0 +1,29 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _server_capability:
+
+**************************
+Manage Server Capabilities
+**************************
+Server capabilities are designed to enable system operators to control the flow of delivery service traffic through caches (Edges or Mids) with ONLY the required capabilities. For example, delivery services that serve large binary files should only be routed to caches with sufficient disk cache. Currently, this can be controlled at the Edge tier where system operators can explicitly assign only Edge caches with sufficient disk cache to the delivery service. However, the system operators do not have control of the Mid tier and cannot dictate which Mid caches are qualified to serve these large binary files. This will cause a problem if a Mid cache with insufficient disk cache is asked to serve the large binary files.
```

Comment:
You've got a bunch of unlinked `term`s in here: "delivery service", "cache" (should be "cache server" in many cases), maybe a couple of others.
Verbal stress on a word in a sentence should be indicated with emphasis roles. Typically you'd do that with asterisks:
```rst
*this text is emphasized*
```
What's an "operator"? I think you mean
```rst
... users with the "operator" :term:`Role` ("operators") ...
```